fastscanner/tests/test_eodhd_fundamental_store.py

```diff
@@ -0,0 +1,204 @@
+import json
```

Comment:
Let's add one or two tests for the public holidays store too in a separate test file